### ISSUE TITLE
Add certificate information in Timeline

### ISSUE THREAD

**Issue by mattisking** [usebruno/bruno#3437]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

The Timeline feature is fine for most needs, but it falls short of Postman's Developer Console. If I'm testing something around security, let's say your organization uses MTLS, and your calls keep coming back with a 403 (or 401), the first thing I check is to verify that my certificate was sent and in more complex collections, that I sent the right certificate. 

I'd like to see the certificate sent in the request to verify the certificate I configured was sent (or it's the one I expected it to be).

### Mockups or Images of the feature

![image001](https://github.com/user-attachments/assets/dc28e6f6-4722-47ab-bb64-6bc261903204)
This is the image from Postman, Developer Console. If a Client Certificate was sent, it will be in the image with the ability to view the file sent, as well as any other cert information. I would expect to see this in the regular Timeline, not in the Bruno Gold Edition Console which is more about Bruno itself. 

I looked at seeing how difficult this would be to do since you pass these neat request and response objects all over the place, but the request doesn't include the certificate information. I don't have much Electron experience, but it appears you are using an ipc call where electron itself is making the match based on the collection configuration and that simply isn't stored in the object. What if you moved that logic back into the site code and stored it in the request, allowing that to be available in the Timeline page (and then electron would just use the cert you matched internally. That, or perhaps attach the certificate selected in the electron app to the response object before passing it back.

**Comment by Its-treason** [comment]
If you need this feature now you can checkout my fork: https://github.com/Its-treason/bruno/releases/tag/nightly It has this feature (Be sure to check `Use new Request method in the Top`) and is fully compatible with Bruno.

**Comment by mattisking** [comment]
You've got quite the series of updates. Thanks. Are some of these changes coming back here? I have no experience with pnpm but that was far faster.

**Comment by Its-treason** [comment]
> You've got quite the series of updates. Thanks. Are some of these changes coming back here? I have no experience with pnpm but that was far faster.

Yes, I have opened [a few pull requests](https://github.com/usebruno/bruno/pulls?q=is%3Apr+author%3AIts-treason), so some fixes will hopefully get merged. I don't expect other things like the new Timeline to be merged, because that would take significant changes in Bruno.

**Comment by mattisking** [comment]
I dunno. Moving that into your Bruno-core and rewriting the request mechanism is exactly what I saw as the thing to do as well. Some of the other things which I thought were quite nice (modals) would probably be tougher but this is exactly what I was looking for.

**Comment by duyn9uyen** [comment]
+1 on needing the client certification information available! I was hoping this was added with Bruno v3. I am moving away from Postman and that is a frequent feature I use to troubleshoot 403 errors. Would love this added to Bruno too.

**Comment by mattisking** [comment]
I believe some of that is there now. If you dig into the details of the timeline you’ll see some details like the certificate subject etc. The helpful addition would be the friendly name.

**Comment by duyn9uyen** [comment]
The Timeline Network Logs only shows the server's certificate information. It doesn't show the client cert that was sent with the HTTPS request.
